fimsauthz-api/fims-authz.yaml

```diff
@@ -70,9 +72,26 @@ paths:
                     The requested scope(s), a list of space-delimited,
                     case-sensitive names. Same as in RFC 6749.
 
-                    Version 1.0.2 of this specificaiton allows a pair of scope for this case only:
+                    As of Version 1.0.2 of this specificaiton allows a pair of scope for this case only:
                     [dss.read.identification_service_areas, rid.read.enhanced_details]
                   type: string
+                resource:
```

Comment:
`resource` is fine, but it seems like there are a lot of types of resources; why not make this specific like `audience` or `requested_audience` or `aud` or `recipient`?

Reply:
We went back and forth a little.  This is a very underspecified thing for some reason in the wild.  We wanted to avoid `aud` even though it typically maps directly to the `aud` claim, just felt better to keep them separate names.

My gut reaction would have been `audience` as well. 

After finding no guidance in any RFC, we went with the suggestion from a well-written blog post:  https://medium.com/@robert.broeckelmann/oauth2-access-token-usage-strategies-for-multiple-resources-apis-part-2-304bab2570a9

Reply:
IMO  nice suggestions Ben, since these names are self-documenting.  Someone gets to pick.

Reply:
That article does seem good, but I'm not fully convinced the term "resource" is needed in order to follow the spirit of the post.  Nonetheless, the name here doesn't actually matter and `resource` is fine if preferred.